<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTranusDialogBase</class>
 <widget class="QDialog" name="QTranusDialogBase">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTranus</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/plugins/QTranus/icon.png</normaloff>:/plugins/QTranus/icon.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Project Configuration: &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;QTRANUS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_help">
       <property name="text">
        <string notr="true">Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeView" name="scenarios"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Project file:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLineEdit" name="layers_group_name">
             <property name="minimunWidth" stdset="0">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="db_btn">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Tranus workspace:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <layout class="QHBoxLayout" name="nameLayout">
           <item>
            <widget class="QLineEdit" name="tranus_folder">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tranus_folder_btn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Zones shape file:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="zone_shape">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="zones_shape_btn">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lbl_shape_fields">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Select zones shape Id:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="cb_zones_shape_fields">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Centroid shape file:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="centroid_shape">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="centroid_shape_btn">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Network links shape file:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="network_links_shape">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="network_links_shape_btn">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Network nodes shape file:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLineEdit" name="network_nodes_shape">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="network_nodes_shape_btn">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="lbl_links_id">
           <property name="text">
            <string>Select links shape Id</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="links_shape_fields"/>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="lbl_nodes">
           <property name="text">
            <string>Select nodes shape Id</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QComboBox" name="nodes_shape_fields"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item alignment="Qt::AlignVCenter">
          <widget class="QCommandLinkButton" name="data">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Data</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QCommandLinkButton" name="run">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Run</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignVCenter">
          <widget class="QCommandLinkButton" name="results">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Results</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lbl_load">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Cargando...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="pg_loading">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="button_box">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Close</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_help</tabstop>
  <tabstop>scenarios</tabstop>
  <tabstop>layers_group_name</tabstop>
  <tabstop>tranus_folder</tabstop>
  <tabstop>tranus_folder_btn</tabstop>
  <tabstop>zone_shape</tabstop>
  <tabstop>zones_shape_btn</tabstop>
  <tabstop>cb_zones_shape_fields</tabstop>
  <tabstop>centroid_shape</tabstop>
  <tabstop>centroid_shape_btn</tabstop>
  <tabstop>network_links_shape</tabstop>
  <tabstop>network_links_shape_btn</tabstop>
  <tabstop>network_nodes_shape</tabstop>
  <tabstop>network_nodes_shape_btn</tabstop>
  <tabstop>data</tabstop>
  <tabstop>run</tabstop>
  <tabstop>results</tabstop>
  <tabstop>button_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>QTranusDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>QTranusDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>605</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
